packages/table/src/Cell/HeaderCell/HeaderCell.spec.tsx

```diff
@@ -198,4 +199,56 @@ describe('packages/table/HeaderCell', () => {
     expect(ref.current).toBeInTheDocument();
     expect(ref.current!.textContent).toBe('Hello');
   });
+
+  describe('styled', () => {
+    test('works with `styled`', () => {
+      const StyledHeaderCell = styled(HeaderCell)`
+        color: #69ffc6;
+      `;
+
+      const { getByTestId } = render(
+        <StyledHeaderCell data-testid="styled">Some text</StyledHeaderCell>,
+      );
+
+      expect(getByTestId('styled')).toBeInTheDocument();
```

Comment:
Nit: Wonder if it's necessary to check if this is in the document? It's not actually an implementation detail since we're rendering it directly in the test. Won't comment this on each one to bloat the review, but have the same thinking for each of these styled tests.